index.bs

```diff
@@ -147,11 +145,80 @@ In general, it is recommended to use properties instead of [=Metadata OBUs=] in
 
 <p>In [[!ISOBMFF]] or [[!HEIF]] files, if an [=AV1 Alpha Image Item=] (respectively. an [=AV1 Alpha Image Sequence=]) is used, the <code>aux_type</code> field of the <code>AuxiliaryTypeProperty</code> (respectively <code>AuxiliaryTypeInfoBox</code>) SHALL be set to <code>urn:mpeg:mpegB:cicp:systems:auxiliary:alpha</code>, as defined in [[!MIAF]].</p>
 
-<h2 id="AVIF-baseline-profile">MIAF AV1 Baseline Profile</h2>
+  <h2 id="brands-and-file-extensions">Brands and file extensions</h2>
+
+  <h3 id="AVIF-general-brand">AVIF General Brand</h3>
+
+Files conformant with the brand-independent restrictions in this document shall include the brand
+<dfn value="" export="" for="AVIF General brand">avif</dfn> in the [=compatible_brands=] field of the [=FileTypeBox=].
+  Files shall also carry a compatible brand to identify the AVIF profile, if any, with which the file complies.
+
+  <h3 id="image-and-image-collection-files">AV1 image and image collection files</h3>
+  The file extension ".avif" and the MIME Content Type "image/avif" should be used for AVIF files that satisfy the following requirements:
+  - The file does not contain tracks, including image sequences, of any kind.
+  - The [=primary image=] is either an [=AV1 Image Item=] or a derived image that references one or more coded images that all are [=AV1 Image Items=].
+  - The AVIF file conforms to at least one AVIF profile, and the profile is listed in the [=compatible_brands=] field of the [=FileTypeBox=].
```

Comment:
Why? If the "Professional" profile is not formally defined in AVIF but one wants to create an AVIF file with a "Professional" image in it, they would not be allowed to use this general brand. That does not seem right.

Reply:
This text is appearing in the section that defines the rules for the ".avif" file extension. It says that if you use the ".avif" file extension, the file needs to have a Primary Image that matches one of the profiles in this spec. It does not prohibit the use AV1 Professional Profile in a file that uses the 'avif' brand. But such a file shouldn't use the ".avif" file extension. It can use the more general ".heif" or ".hif" file extensions instead.

Reply:
I did understand that you had a comment on that (maybe it was a different comment?) The idea is that if you have a file that uses the file extension for still image files, there should not be image tracks in that file, because there is a separate file extension for that. The idea was to have a rule to force people to use the correct file extension for image-track files.
But I agree that having a restrictive rule like this does not follow established pattern, so I have already removed this sentence from the second iteration that you are reviewing now.